.travis.yml

```diff
@@ -6,7 +6,7 @@ install:
   - npm install
 
 branches:
-  only: 
+  only:
```

Comment:
@twrdyyy please avoid adding unnecessary whitespaces in the future as it only creates noise inside diffs.